libs/clients/national-registry/v3/src/lib/apiConfig.ts

```diff
@@ -0,0 +1,38 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+
+import { Configuration } from '../../gen/fetch'
+import { NationalRegistryV3ClientConfig } from './nationalRegistryV3.config'
+
+export const ApiConfig = {
+  provide: 'NationalRegistryV3ClientProviderConfiguration',
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xroadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof NationalRegistryV3ClientConfig>,
+  ) =>
+    new Configuration({
+      fetchApi: createEnhancedFetch({
+        name: 'clients-national-registry-v3',
+        timeout: config.fetchTimeout,
+        autoAuth: {
+          mode: 'token',
+          clientId: config.clientId,
+          clientSecret: config.clientSecret,
+          scope: [config.scope],
+          issuer: '',
```

Comment:
The token endpoint doesn't follow the '{issuer}/connect/token' so we have to use the tokenEndpoint property instead of the usual. However, the issuer is a required prop so we just provide an empty string